build.sbt

```diff
@@ -7,19 +7,47 @@ ThisBuild / scmInfo := Some(
   ScmInfo(url("https://github.com/avast/scala-server-toolkit"), "scm:git:git@github.com:avast/scala-server-toolkit.git")
 )
 
-ThisBuild / scalaVersion := "2.13.1"
+ThisBuild / scalaVersion := "2.13.0"
 ThisBuild / scalacOptions := ScalacOptions.default
 
+ThisBuild / turbo := true
+
 lazy val commonSettings = Seq(
   libraryDependencies ++= Seq(
-    compilerPlugin("org.typelevel" %% "kind-projector" % "0.10.3")
+    compilerPlugin("org.typelevel" %% "kind-projector" % "0.10.3"),
+    "org.typelevel" %% "cats-effect" % "2.0.0",
+    "org.scalatest" %% "scalatest" % "3.0.8" % Test
   ),
   Test / publishArtifact := false
 )
 
 lazy val root = (project in file("."))
+  .aggregate(example, pureconfig)
   .settings(
-    commonSettings,
     name := "scala-server-toolkit",
     publish / skip := true
   )
+
+lazy val example = project
+  .dependsOn(pureconfig)
+  .enablePlugins(MdocPlugin)
+  .settings(
+    commonSettings,
+    name := "scala-server-toolkit-example",
+    publish / skip := true,
+    run / fork := true,
+    Global / cancelable := true,
+    mdocIn := baseDirectory.value / "src" / "main" / "mdoc",
+    mdocOut := baseDirectory.value / ".." / "docs",
+    libraryDependencies ++= Seq(
+      "dev.zio" %% "zio" % "1.0.0-RC12-1",
+      "dev.zio" %% "zio-interop-cats" % "2.0.0.0-RC3"
+    )
+  )
+
+lazy val pureconfig = project
+  .settings(
+    commonSettings,
+    name := "scala-server-toolkit-pureconfig",
+    libraryDependencies += "com.github.pureconfig" %% "pureconfig" % "0.12.0"
```

Comment:
Since most of the submodules will usually depend only on single dependency (which will not be shared with any other module) I don't see the benefit - this way I see directly what depenedency and in which version is used. So I'd like to keep it that way. OK?

Reply:
I don't think it's true. Non trivial amount of deps will at least depend on cats-effect.

Reply:
Yes, `cats-effect` is the only exception and that's why it's part of `commonSettings` shared by all the modules (and defined in single place).

Reply:
I still thing it should be extracted into separate file as it is more manageable. I won't block it right now, but mark my words that we will move it to separate file eventually.

Reply:
Alright, I did it ;)